app/Http/Controllers/Auth/ChangePasswordController.php

```diff
@@ -24,8 +24,12 @@ public function edit()
 
     public function update(UpdatePasswordRequest $request)
     {
-	$info = $request->validated();
-	$info['password_changed_at'] = Carbon::now();
+        $info = $request->validated();
+        $info['updated_at'] = Carbon::parse();
+	    $info['password_changed_at'] = Carbon::now();
```

Comment:
password_changed_at must be taken from that user's table row not the current date